test/NuGet.Core.FuncTests/Dotnet.Integration.Test/MsbuildIntegrationTestFixture.cs

```diff
@@ -208,6 +212,28 @@ private CommandRunnerResult PackProjectOrSolution(string workingDirectory, strin
             return result;
         }
 
+        private CommandRunnerResult PackProjectOrSolutionWithRepoPackArtifacts(string workingDirectory, string file, string args, string nuspecOutputPath, bool validateSuccess)
```

Comment:
Why do we need this?
What does this achieve over the current methods? They seem equivalent but maybe I am missing something.

Reply:
The pack will use the NuGet.Build.Tasks.Pack.dll from the repo artifacts.

Reply:
The pack SDK has already been copied to .NET CLI. 
This is what the fixture does, runs the constructor and copies built artifacts into the CLI folder.

Reply:
A comment above - reverted MsbuildIntegrationTestFixture changes.